tests/vmi_multus_test.go

```diff
@@ -658,6 +668,19 @@ var _ = Describe("SRIOV", func() {
 			return
 		}
 
+		createSRIOV_VMI := func(networksNames []string) (vmi *v1.VirtualMachineInstance) {
```

Comment:
variables in golang are not snake cases (a_b), but usually camel case.

For consistency sake please rename `createSRIOV_VMI` to `createSriovVMI` . 

You can skip this suggestion, and we can wait to get a maintainer's opinion.